docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3.9'
+
+services:
+  php-8.0: &php
```

Comment:
wouldn't it be better `php80`? so as to make it easier to use from the cli with `docker-compose exec php80 sh`?

Reply:
Ok i'll change it in next commit

Reply:
Done @JellyBellyDev